src/lib/components/ApplicationRow.svelte

```diff
@@ -1,27 +1,29 @@
 <script lang="ts">
 	import * as modal from '$lib/utils/modal';
-	import Tag from '$lib/shared/Tag.svelte';
-	import User from '$lib/shared/User.svelte';
-	import ApplicationDetail from '@components/ApplicationDetail/index.svelte';
+	import User from './User.svelte';
+	import ApplicationDetail from './ApplicationDetail.svelte';
+	import type { Application } from '$lib/types';
 
-	export let owner: boolean = false;
```

Comment:
Why remove the type declaration if it's a typescript file?

Reply:
As `owner` is already declared as a boolean literal it is already of type boolean so we don't need the type declaration here.

This also applies for numbers and strings.